### ISSUE TITLE
tox 3.4 breakage: NoneType is not iterable

### ISSUE THREAD

**Issue by asottile** [tox-dev/tox-pip-extensions#16]
```
$ tox -e py37
Traceback (most recent call last):
  File "/home/asottile/bin/tox", line 11, in <module>
    sys.exit(cmdline())
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/tox/session.py", line 41, in cmdline
    main(args)
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/tox/session.py", line 46, in main
    config = prepare(args)
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/tox/session.py", line 28, in prepare
    config = parseconfig(args)
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/tox/config.py", line 233, in parseconfig
    pm.hook.tox_configure(config=config)  # post process config object
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/pluggy/hooks.py", line 258, in __call__
    return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/pluggy/manager.py", line 67, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/pluggy/manager.py", line 61, in <lambda>
    firstresult=hook.spec_opts.get('firstresult'),
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/pluggy/callers.py", line 201, in _multicall
    return outcome.get_result()
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/pluggy/callers.py", line 76, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/pluggy/callers.py", line 180, in _multicall
    res = hook_impl.function(*args)
  File "/home/asottile/opt/venv/lib/python3.6/site-packages/tox_pip_extensions.py", line 48, in tox_configure
    cfg.read(config.option.configfile)
  File "/usr/lib/python3.6/configparser.py", line 694, in read
    for filename in filenames:
TypeError: 'NoneType' object is not iterable
```

**Comment by asottile** [comment]
CC @gaborbernat

**Comment by asottile** [comment]
@gaborbernat any suggestions on how to access the `[tox]` config namespace now that `config.option.configfile` is ~usually `None`?

**Comment by asottile** [comment]
ah, seems I can use `config.toxinipath` instead of `config.option.configfile`

**Comment by asottile** [comment]
CC @seanh -- this has been released as v1.4.1

**Comment by seanh** [comment]
Works here! Thanks for the cc, I managed to miss it in my email for three days
